SimpleCalculator.py

```diff
@@ -90,12 +90,17 @@ def calculate_average(values):
 # TODO this function should take list of values,
 # then returns sorted list in ascending(normal) or descending(reverse" order
 def sort_values(values, order="normal"):
-    for x in range(0,5):
-        for i in range(0,5):
+    for x in range(len(values)):
+        for i in range(5):
```

Comment:
You could do the same as for code above, I think this should work fine, you know "range(len(values)), I think it should work fine